contrib/tune-spec.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+PRE_VERSION=${1:-}
+VERSION=${2:-}
+RELEASE=${RELEASE:-1}
+SPEC_FILE=${SPEC_FILE:-cn.spec}
+NOW=$(LANG=C date "+%a %b %d %Y")
+
+function fatal() {
+    echo "$@"
+    exit -1
+}
+
+
+if [ ! -f $SPEC_FILE ]; then
+    fatal "$SPEC_FILE doesn't exists !"
+fi
+
+if [ -z "$PRE_VERSION" ]; then
+    fatal "PRE_VERSION shall be defined !"
+fi
+
+if [ -z "$VERSION" ]; then
+    fatal "VERSION shall be defined !"
+fi
+
+RPM_VERSION=$(echo ${VERSION//v/})
+TEMP=$(mktemp /tmp/tunespec.XXXXXXXXX)
+START=$TEMP.start
+CHANGELOG=$TEMP.changelog
+
+CHANGELOG_POS=$(grep -n %changelog $SPEC_FILE | cut -d ":" -f 1)
+
+#Extrating the header of the spec file
+head -$CHANGELOG_POS $SPEC_FILE > $TEMP
```

Comment:
[shellcheck] Double quote to prevent globbing and word splitting. [SC2086] [2086]